plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/grouped-product-selector/product-item-template/block.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "woocommerce/add-to-cart-with-options-grouped-product-selector-product-item-template",
+	"version": "1.0.0",
+	"title": "Grouped Product Selector Item Template (Experimental)",
+	"description": "A product item template.",
+	"category": "woocommerce-product-elements",
+	"keywords": [ "WooCommerce" ],
+	"usesContext": [ "postId" ],
+	"ancestor": [ "woocommerce/add-to-cart-with-options" ],
+	"textdomain": "woocommerce",
+	"apiVersion": 3,
+	"$schema": "https://schemas.wp.org/trunk/block.json"
```

Comment:
We will probably want to add `"inserter": false` into this block so it doesn't appear in the block inserter.

Reply:
done -> [c76532f](https://github.com/woocommerce/woocommerce/pull/54651/commits/c76532fb35899d63827e546a1fca05c6fbb06e6a)